src/views/MessagePage.tsx

```diff
@@ -0,0 +1,185 @@
+import React, { useState, useEffect } from 'react';
+import { RouteComponentProps } from 'react-router';
+
+import { Typography, makeStyles, TextField, List, ListSubheader, ListItem, ListItemText } from '@material-ui/core';
+
+import { inject, observer } from 'mobx-react';
+import { FILE_STORE, IFileStore } from '../stores/fileStore';
+
+const useStyles = makeStyles(theme => ({
+  container: {
+    display: 'grid',
+    height: '100vh',
+    gridTemplateColumns: '1fr 2fr 1fr 1fr',
+    gridTemplateRows: 'auto 3fr 4fr',
+    // gridTemplateAreas: `
```

Comment:
Maybe we can delete commented out code if we don't need it?

Reply:
yeah I was playing around with different layouts and allowing resizing, but this is easy to recreate if we want to adjust layout in the future, I'll delete it.